### PR TITLE
Zbs280/replace docker entrypoint

### PR DESCRIPTION
**Pull request by zack-erisyon**
This is a significant cleanup of legacy "p" code moving the dev features to h.
I'd like to get feedback from everyone as there are possible workflows that I don't know about.

### CODE REVIEW

File: scripts/docker_entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+from plumbum import cli, local
+
+
+class DockerApp(cli.Application):
+    def main(self):
+        print("In plumbum!")
+
+
+if __name__ == "__main++":
```
Comment (ossareh) [inline scripts/docker_entrypoint.py:11]: I've not seen the `__main++` form before, how does it differ from `__main__`?

**Review by ossareh — COMMENTED** [review]
you said that stuff moved from `p` to `h`, but I can't see those changes here. Am I missing something?

**Comment by ossareh** [comment]
oh, I see that this is paired with a change to our internal repo. That was not clear.

File: scripts/docker_entrypoint.py (hunk 1 above)
Comment (danwinkler) [reply scripts/docker_entrypoint.py:11]: This has got to be a typo, right?

Comment (ossareh) [reply scripts/docker_entrypoint.py:11]: That was my first assumption, but I also assumed that Zack ran this code 🤷🏻‍♂️ - perhaps it wasn't run to catch bugs like this?

**Pull request merged**